### DISCUSSION TITLE
Plugins for Leantime

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#1373] [Plugins]
As of 2.3 Leantime it is possible to write plugins for Leantime. We added a sample plugin to the repository. 
I am still working on a more in depth documentation but I'd love to start the discussion on possible plugins we should create.

**Comment by kav2k** [comment]
For me, the most obvious thing to look for in a plugin system is capability for more integrations, e.g. capability to solve #354 in a plugin

**Comment by kav2k** [comment]
Another decent idea are auth modules, to support more SSO scenarios

**Comment by neurabluAI** [comment]
For us it would be great to have something similar to Draw.io since we would like to draw architecture overviews in the Wiki Section.

**Reply by samuk** [reply]
https://www.tldraw.com/ might be a nice option

**Reply by neurabluAI** [reply]
Nice one. Thank you for the link. Much appreciated

**Reply by DimitReeis** [reply]
Excalidraw is also an option

**Comment by brayleB** [comment]
Hello, do you have idea how to add Adyen plugin in here?

**Comment by Snuffsis** [comment]
One suggestion for a plugin that would be nice to have official support for would be a github integration.
So you could link projects with repos and todos with issues.

**Reply by gloriafolaron** [reply]
Github integration is definitely on our to come as soon as we can list -- thanks for calling it out. 

As people make requests for those things, it'll get easier for us to prioritize them OR even allow community members to contribute those as well.

**Comment by marcelfolaron** [comment]
We have a whiteboard plug-in now.

**Comment by lukstruck** [comment]
I'm thinking of building an automations plugin, i.e. that all tasks added in a certain list automatically get their due date set to in 2 days or something. Or when they are moved to done, the due date is set to right now, so that they dont show anymore in the calender.

**Comment by lukstruck** [comment]
A Github integration. (symmetric if possible) when an issue gets created, a task gets created in leantime. Descriptions are synced (last write wins is enough) both ways. Comments are synced. Status gets synched.

**Comment by lukstruck** [comment]
A mail to task integration. Lets me connect an imap server, (has to have catch all). I can generate a mail address for a list, and when I send a mail to that address, it gets added in that list as a task (like asana, trello, etc...)
